grafanalib/core.py

```diff
@@ -299,6 +302,61 @@ def TimePicker(refreshIntervals, timeOptions):
 )
 
 
+def AlertCondition(target, queryParams=["A", "5m", "now"],
+                   evaluator=["gt", 5], operator="and", type="avg"):
+    """
+    Alert conditions:
+        :param target: Target
+        :param queryParams: [refId, in_time, "now"]
+        :param evaluator: [gt/lt/within_range/outside_range/no_value, values]
+        :param operator: and/or
+        :param type: avg/min/max/sum/count/last/median
+    """
+    evaluator_params = []
+    if evaluator[0] in ['lt', 'gt']:
+        evaluator_params = [evaluator[1]]
+    elif evaluator[0] in ['outside_range', 'within_range']:
+        evaluator_params = [evaluator[1], evaluator[2]]
+    return {
+        "evaluator": {
+            "params": evaluator_params,
+            "type": evaluator[0]
+        },
+        "operator": {
+            "type": operator
+        },
+        "query": {
+            "model": target,
+            "params": queryParams
+        },
+        "reducer": {
+            "params": [],
+            "type": type
+        },
+        "type": "query"
```

Comment:
Trailing comma please, and also probably make `type` a parameter that defaults to `"query"`.